catroid/src/main/res/values-ar/strings.xml

```diff
@@ -49,19 +49,19 @@
   <string name="close">إغلاق</string>
   <string name="error">خطأ</string>
   <string name="please_wait">الرجاء الانتظار</string>
-  <string name="loading">تحميل&#8230;</string>
-  <string name="search_hint">جاري البحث&#8230;</string>
+  <string name="loading">تحميل&#x2026;</string>
+  <string name="search_hint">جاري البحث&#x2026;</string>
   <string name="no_internet">لا يوجد إتصال بالإنترنت</string>
-  <string name="loading_check_token">Token is checked on our server&#8230;</string>
-  <string name="loading_check_facebook_data">يتم استرداد بيانات الفيس بوك&#8230;</string>
-  <string name="loading_check_email">E-Mail availability is checked on our server&#8230;</string>
-  <string name="loading_check_oauth_token">OAuth token is checked on our server&#8230;</string>
-  <string name="loading_check_facebook_token_validity">Facebook token validity is checked on our server&#8230;</string>
-  <string name="loading_check_oauth_username">Username availability is checked on our server&#8230;</string>
-  <string name="loading_facebook_exchange_token">Facebook token is exchanged on our server&#8230;</string>
-  <string name="loading_facebook_login">Facebook login is completed on our server&#8230;</string>
-  <string name="loading_google_exchange_code">تم تبادل جوجل + التعليمات البرمجية على الخادم&#8230;</string>
-  <string name="loading_google_login">تم إكمال عملية جوجل +تسجيل الدخول على الخادم&#8230;</string>
+  <string name="loading_check_token">Token is checked on our server&#x2026;</string>
+  <string name="loading_check_facebook_data">يتم استرداد بيانات الفيس بوك&#x2026;</string>
+  <string name="loading_check_email">E-Mail availability is checked on our server&#x2026;</string>
+  <string name="loading_check_oauth_token">OAuth token is checked on our server&#x2026;</string>
+  <string name="loading_check_facebook_token_validity">Facebook token validity is checked on our server&#x2026;</string>
+  <string name="loading_check_oauth_username">Username availability is checked on our server&#x2026;</string>
+  <string name="loading_facebook_exchange_token">Facebook token is exchanged on our server&#x2026;</string>
+  <string name="loading_facebook_login">Facebook login is completed on our server&#x2026;</string>
+  <string name="loading_google_exchange_code">تم تبادل جوجل + التعليمات البرمجية على الخادم&#x2026;</string>
```

Comment:
those are right to left languages, so you added the "..." to the begin of the sentence...